<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RankingListWindow</class>
 <widget class="QMainWindow" name="RankingListWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="returnToMenuButton">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>600</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>返回主菜单</string>
    </property>
   </widget>
   <widget class="QWidget" name="rankingListWidget" native="true">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>110</y>
      <width>668</width>
      <height>450</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="rankingListVerticalLayout">
       <item>
        <widget class="QTableWidget" name="rankingListTableWidget"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="rankingListLabel">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>35</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>排行榜</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
